### PR TITLE
SEO - 171884 - Page with Redirect Fix 1

### PR DESCRIPTION
**Pull request by Thahira-Banu**
Hi @Aishwarya-Ganesan 

I have made changes in the Page with Redirect Links by replacing redirect links.

Thanks & Regards

### CODE REVIEW

File: uwp/RichTextBox/Comment.md [hunk 1]
```diff
@@ -87,3 +87,22 @@ private void RichTextBoxAdv_CommentAdding(object obj, CommentAddingEventArgs arg
 {% endtabs %}
 ![Comment_img2](Comment_images/Comment_img2.jpeg)
 
+## Visibility of comment pane
```
Comment (MallikaRK) [inline uwp/RichTextBox/Comment.md:90]: Are you made this changes?

**Review by MallikaRK — CHANGES_REQUESTED** [review]

File: uwp/Charts/Series-old.md [hunk 2]
```diff
@@ -0,0 +1,4302 @@
+---
```
Comment (MallikaRK) [inline uwp/Charts/Series-old.md:1]: Why do you create duplicate series.md file. Check and revert it.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Comment by Thahira-Banu** [comment]
I have made changes in multiple PR's, so closed it.

**Pull request closed without merging**